### ISSUE TITLE
ID Anchor Link Issues

### ISSUE THREAD

**Issue by Vishkujo** [StarCitizenTools/mediawiki-extensions-TabberNeue#88]
When linking to an ID anchor link of a div that is inside a tab, it removes everything else from the tab.

Here's an [example](https://jojowiki.com/Super_Action_Statue#Crazy_Diamond_Second). It removed the "Crazy Diamond" box above "Crazy Diamond Second", and only shows the latter one while leaving a large gap underneath it.

**Comment by alistair3149** [comment]
Is this still reproducible? I don't see the issue on the site.

**Comment by alistair3149** [comment]
Hi sorry for getting back late. Does it still happen in the latest version?

**Comment by Vishkujo** [comment]
Yeah, I still get the issue. I tested it on the Star Citizen wiki just to make sure it wasn't my CSS or JavaScript causing the issue.
**Steps to reproduce:**
1. Check this [page](https://starcitizen.tools/User:Vish)
2. Click the "Crazy Diamond" tab
3. Click "Expand" on either the "Crazy Diamond" box's Gallery or the "Crazy Diamond Second" box's Gallery. You'll see that when the gallery expands, the boxes don't fit within the tabber panel anymore (the bottom gets cut off). 
4. Try clicking "Link to this section" on the "Crazy Diamond Second" box. The panel will scroll to the bottom, cutting off the top of the first box. (this one doesn't seem to happen on Chrome, the anchor does nothing there for some reason, but it happens on Firefox)

[Video example](https://i.imgur.com/y2NSP8b.mp4)

It seems like the problem might be related to the height of tabber__panel. It might need to recalculate height if the content inside it changes?

**Comment by alistair3149** [comment]
Ooooh I finally get it. Yes it is a known issue that I have been trying to fix over the last few weeks. Somehow the resize observer is not working correctly on the tabpanels, which prevents the tabpanels changing its height according to the content. I'll give it another try next week and see if I can patch it.

**Comment by alistair3149** [comment]
Can you still reproduce the issue now with the latest version?

**Comment by KidaJiwo** [comment]
Currently I can reproduce this bug (or a different, related one that causes the same effect) by following the following steps, using Vishkujo's [previously posted page](https://starcitizen.tools/User:Vish):

1. Switch to "Crazy Diamond" tab
2. Click on "Expand" on both "Gallery" sections
3. Switch back to "Josuke Higashikata" tab
4. Click on the second "Link to this section" link under the "Josuke Higashikata Second" heading

[Video example](https://files.catbox.moe/lg03gt.webm). 
This will cause the top of the first section to be cut off. This happens on both Firefox and Chrome browsers.

----
A very similar issue relating to References IDs occurs on our Wiki, though I should mention we are running TabberNeue version 2.7.2, since we are running MediaWiki on the 1.39.11 Maintenance version, where using the newest version of TabberNeue currently breaks all tabbers.
Visiting [this page](https://orangejuice.wiki/wiki/Alte), and following the next steps:

1. Switch to "Trivia" tab
2. Under "References", click on the "↑" symbol (hovering will show "Jump up"). 
  Really noticeable clicking on the 3rd one, "`100% Web RadiOrange Juice #3 with Hono`".

[Video example](https://files.catbox.moe/58xnc6.webm).
Will cause a very similar issue, cutting off everything above the line where the citation is.
